wordle-web/pages/dailyGame.vue

```diff
@@ -0,0 +1,233 @@
+                    fab
+                    v-bind="attrs"
+                    v-on="on"
+                  >
+                    <v-icon>mdi-home</v-icon>
+                  </v-btn>
+                </v-row>
+              </v-container>
+            </template>
+            <span> Go Home </span>
+          </v-tooltip>
+        </v-col>
+        <v-col cols="12" sm="5" class="d-flex flex-row-reverse">
+          <v-dialog v-model="dialog" justify-end persistent max-width="600px">
+            <template #activator="{ on, attrs }">
+              <v-btn color="primary" dark v-bind="attrs" v-on="on">
+                {{ playerName }}
+              </v-btn>
+            </template>
+            <v-card>
+              <v-card-text>
+                <v-text-field
+                  v-model="playerName"
+                  type="text"
+                  placeholder="Guest"
+                  color="#3A96B5"
+                ></v-text-field>
+              </v-card-text>
+              <v-card-actions>
+                <v-spacer></v-spacer>
+                <v-btn color="blue darken-1" text @click="dialog = false">
+                  Close
+                </v-btn>
+                <v-btn
+                  color="blue darken-1"
+                  text
+                  @click=";(dialog = false), setUserName(playerName)"
+                >
+                  Save
+                </v-btn>
+              </v-card-actions>
+            </v-card>
+          </v-dialog>
+        </v-col>
+      </v-row>
+      <v-row>
+        <v-col cols="12" sm="3"></v-col>
+        <v-col
+          cols="12"
+          sm="6"
+          class="mt-0 mb-0 pt-0 pb-0 text-center display-2"
+        >
+          DAILY<NotWordleLogo />
+        </v-col>
+        <v-col cols="12" sm="3">
+          <v-card-text align="right">
+            <v-icon>mdi-timer</v-icon>
+            {{ displayTimer() }}
+          </v-card-text>
+        </v-col>
+      </v-row>
+      <v-row justify="center" class="mt-10">
+        <v-alert v-if="wordleGame.gameOver" width="80%" :type="gameResult.type">
+          {{ gameResult.text }}
+          <v-btn class="ml-2" @click="resetGame"> Play Again? </v-btn>
+          <v-btn class="ml-2" nuxt to="/game"> Play Unlimited? </v-btn>
+        </v-alert>
+      </v-row>
+
+      <v-row justify="center">
+        <game-board :wordleGame="wordleGame" />
+      </v-row>
+      <v-row justify="center">
+        <keyboard :wordleGame="wordleGame" />
+      </v-row>
+    </v-container>
+    <v-col>
+      <v-card-title>
+        <Birdle-icon />
+      </v-card-title>
+    </v-col>
+  </v-app>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { GameState, WordleGame } from '~/scripts/wordleGame'
+import KeyBoard from '@/components/keyboard.vue'
+import GameBoard from '@/components/game-board.vue'
+import { Word } from '~/scripts/word'
+
+@Component({ components: { KeyBoard, GameBoard } })
```

Comment:
Alot of the code in this typescript could probably be wrapped in a service and shared between the `dailyGame.vue` page and the `game.vue` page.